standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -129,8 +129,20 @@ protected Connection getConnection() {
     throw new RuntimeException("unexpected type for connection handle");
   }
 
+  protected boolean isOpen() {
+    if (handle == null) {
+      return false;
+    }
+    try {
+      return ((Connection)handle).isClosed();
+    } catch (SQLException e) {
+      LOG.warn("Could not validate jdbc connection to"+ jdbcUrl, e);
```

Comment:
nit: need a space after "to" and before the jdbcUrl
so "to " + jdbcUrl

Reply:
Thx, will fix it.